Packs/ConcentricAI/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,2 @@
+##### ConcentricAI
+- Added File Sharing Details in Playbook along with an additional command.
```

Comment:
Your release note doesn't match the expected standard and is missing release notes for the playbook.
I have added the required format below. Please fill in the missing release note for the playbook.
```suggestion
#### Integrations
##### ConcentricAI
- Added File Sharing Details in Playbook along with an additional command.
#### Playbooks
##### ConcentricAI Demo Playbook
```

Reply:
Done !